### ISSUE TITLE
AlignedStruct support

### ISSUE THREAD

**Issue by waszil** [timrid/construct-typing#9]
Hi! Great package!
I use `AlignedStruct` from construct, is there a way to wrap it by `DataclassStruct`?
Thanks!

**Comment by timrid** [comment]
Yes, this should also be possible in principle. But I have not implemented that yet. I put it on my TODO list, but it will take some time until I will implement it...

As a workaround you can declare each field as `Aligned`. `AlignedStruct` does nothing else, but automated.

**Comment by waszil** [comment]
You're right, using `Aligned` for each field works just fine. Thanks!
